socorro/external/es/create_recent_indices_app.py

```diff
@@ -43,7 +41,7 @@ def main(self):
         today = date.today()
         current_monday = today - timedelta(days=today.weekday())
 
-        for week_diff in range(-1, self.config.elasticsearch_weeks_to_create):
+        for week_diff in range(-2, self.config.elasticsearch_weeks_to_create):
```

Comment:
The "super search missing fields" page looks at the previous two weeks. That fails with an HTTP 500. I figure it's easier to tweak the local development environment so it exists.